scripts/alert_emails/generate_daily_alerts.ts

```diff
@@ -0,0 +1,120 @@
+/**
+ * Generates the daily alert file. Given two snapshots, finds differences in the
+ * overall threat level and each metric and generates a summary file.
+ *
+ * Run via: yarn generate-daily-alerts [old_snapshot new_snapshot]
+ *
+ * If no snapshots provided, uses the master snapshot and the current branch's
+ * snapshot.
+ *
+ * Generates a file alerts/<old_snapshot>-<new_snapshot>.json containing an
+ * array of alerts. See the Alert interface below for schema.
+ */
+
+import fs from 'fs-extra';
+import path from 'path';
+import {
+  fetchAllStateProjections,
+  fetchAllCountyProjections,
+} from '../../src/common/utils/model';
+import { ProjectionsSet } from '../../src/common/models/ProjectionsSet';
+import { Projections } from '../../src/common/models/Projections';
+import { fetchMasterSnapshotNumber, snapshotFromUrl, snapshotUrl } from '../../src/common/utils/model';
+import { SNAPSHOT_URL } from '../../src/api';
+import { Level } from '../../src/common/level';
+import { Metric, ALL_METRICS } from '../../src/common/metric';
+
+const outputFolder = path.join(__dirname, 'alerts');
+
+// Disable counties; mostly useful for faster testing / debugging.
+const STATES_ONLY = false;
+
+interface Alert {
+  fips: string;
+  oldLevel: Level;
+  newLevel: Level;
+  changedMetrics: Array<{
+    metric: Metric,
+    oldLevel: Level,
+    newLevel: Level
+  }>;
+}
+
+async function main() {
+  const {oldSnap, newSnap} = await parseArgs();
+
+  console.log('Fetching projections from snapshots...');
+  const projectionsSet = ProjectionsSet.fromProjections(
+    await fetchAllProjections(snapshotUrl(oldSnap)),
+    await fetchAllProjections(snapshotUrl(newSnap))
+  );
+
+  const alerts: Alert[] = [ ];
+  for(const pair of projectionsSet.pairs) {
```

Comment:
is projectionSet new?

Reply:
Yeeeaaah...  

So I added that Monday when adding county support to the /compare tool for the modeling team.  Apologies that:
1) I didn't realize this would coincide so nicely with the email alerts stuff.  If I had, I would have coordinated with you on it.
2) I added it without code review because /compare is kinda' my baby and nobody else cares about the code.  If you _want_ to review, I'm happy to make changes.  In particular, the naming could probably be better.